src/plugins/auto/plugin.cpp

```diff
@@ -52,11 +52,52 @@ namespace {
         }
         return METRIC_VALUE(FP32);
     }
+    int MapPriorityValues(ov::hint::Priority priority) {
+        switch (priority) {
+            case ov::hint::Priority::HIGH:
+                return 0;
+            case ov::hint::Priority::MEDIUM:
+                return 1;
+            case ov::hint::Priority::LOW:
+                return 2;
+            default:
+                return 1;
+        }
+    }
+    std::map<std::string, std::string> convert_to_string_map(ov::AnyMap& properties) {
```

Comment:
this is to minimize the changes to internal APIs, and also, old plugin and icore implementation are with old string APIs, this can be removed when plugin switched to 2.0 also